manifests/params.pp

```diff
@@ -1,9 +1,17 @@
 class apt::params {
 
-  if $caller_module_name and $caller_module_name != $module_name {
+  if defined('$caller_module_name') and $caller_module_name and $caller_module_name != $module_name {
     fail('apt::params is a private class and cannot be accessed directly')
   }
 
+  if $::osfamily != 'Debian' {
+    fail('This module only works on Debian or derivatives like Ubuntu')
+  }
+
+  $xfacts = {
+    'lsbdistcodename' => defined('$lsbdistcodename') ? { true => $::lsbdistcodename, default => undef },
```

Comment:
I considered it but since this might become a hash of multiple options I figured a one-liner would be clearer eventually. What do we do?